gravitee-apim-console-webui/Dockerfile

```diff
@@ -0,0 +1,24 @@
+#
```

Comment:
### :warning: Aqua detected misconfiguration in your code  
**Misconfiguration ID:** AVD-DS-0026   
**Check Name:** No HEALTHCHECK defined   
**Severity:** LOW   
**Message:** Add HEALTHCHECK instruction in your Dockerfile  
**Resolution:** Add HEALTHCHECK instruction in Dockerfile  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/misconfig/ds026